### PR TITLE
Add validation to ensure slugs are unique

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: app/models/spotlight/exhibit.rb [hunk 1]
```diff
@@ -56,8 +57,6 @@ class Exhibit < ActiveRecord::Base
     after_create :initialize_main_navigation
     include Spotlight::DefaultThumbnailable
 
-    scope :published, -> { where(published: true) }
```
Comment (jcoyne) [inline app/models/spotlight/exhibit.rb]: Why is this change in this PR?

Comment (cbeer) [inline app/models/spotlight/exhibit.rb]: This line is duplicated on line 9. It probably should be in a separate commit, though.
